book/doctrine.rst

```diff
@@ -979,126 +1060,68 @@ To relate the ``Category`` and ``Product`` entities, start by creating a
             </entity>
         </doctrine-mapping>
 
-First, since a ``Category`` object will relate to many ``Product`` objects,
-a ``products`` array property is added to hold those ``Product`` objects.
-Again, this isn't done because Doctrine needs it, but instead because it
-makes sense in the application for each ``Category`` to hold an array of
-``Product`` objects.
+Doctrine does not *require* the "one" side of a one-to-many relationship to
+hold a collection of its "many" related entities. But in the context of our
+application, it makes sense for each ``Category`` object to hold a collection
+of ``Product`` objects. However, if we had decided against adding a ``$products``
+property to the ``Category`` class, then the ``Product`` entity's ``inversedBy``
+metadata would have to be omitted.
```

Comment:
This is the *first* spot where I've found your explanation and wording anything but wonderful. Here, the first sentence isn't completely clear (and as you mentioned, it's very important that we're clear). When you say "does not require" - it's not clear if the mapping is not required (this is what you meant) or if the mapping is required, but "holding a collection of its many related entities" is somehow the optional part.

This is a tricky spot: as it's a "choice", and it's not clear why I would make one choice or another. Let's recommend that they *do* make both sides of the relationship (as we're doing now), and make this more of a light note - e.g. btw, you don't have to map the OneToMany side, but if you're unsure, do it.